packages/manager/modules/emailpro/src/dashboard/translations/Messages_fr_FR.json

```diff
@@ -686,9 +686,14 @@
   "emailpro_tab_domain_diagnostic_spf_toolbox_ok": "Configuration automatique effectuée",
   "emailpro_tab_domain_diagnostic_dkim": "DKIM",
   "emailpro_tab_domain_diagnostic_dkim_title_error": "Diagnostic DKIM",
+  "emailpro_tab_domain_diagnostic_dkim_id_generation_no_ovhcloud": "La génération de vos identifiants sont en cours, veuillez patienter...",
```

Comment:
I think there is a mistake, in my opinion it's the "génération" that is "en cours" and it should be "est en cours".

Reply:
Oups, Sorry I copy/pasted the french text given without checking it!